tests/e2e/lib/setup-env.js

```diff
@@ -166,7 +204,7 @@ async function maybePreConnect() {
 }
 
 // The Jest timeout is increased because these tests are a bit slow
-jest.setTimeout( PUPPETEER_TIMEOUT || 300000 );
+jest.setTimeout( TIMEOUT || 300000 );
```

Comment:
`TIMEOUT` env var seems quite vague. Do you think it won't clash with any other `TIMEOUT`s out there?

Reply:
Sure, I can rename it to be safer. I just don't want to tie it to any tool.